### PR TITLE
included all the requirements for the build of images

### PR DESCRIPTION
**Pull request by harshad16**
included all the requirements for the build of images
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: #2 

## Description

Dockerfile has the binary requirement installation for pytesseract and opencv

### CODE REVIEW

File: .s2i/environment [hunk 1]
```diff
@@ -0,0 +1 @@
+XDG_CACHE_HOME=/opt/app-root/src/.cache
```
Comment (goern) [inline .s2i/environment]: why dont we put this on `/tmp` rather than the container filesystem?

File: overlays/ps-cv-ocr/Pipfile [hunk 2]
```diff
@@ -6,6 +6,11 @@ name = "pypi"
 [packages]
 opencv-python = "*"
 pytesseract = "*"
+pillow = "*"
+scikit-image = "*"
+numpy = "*"
```
Comment (pacospace) [inline overlays/ps-cv-ocr/Pipfile]: Why are we adding packages already present in the base DS generic image?

**Review by pacospace — APPROVED** [review]
Thanks! One last comment is for scikit image, also part of the ds generic image actually, but lets go ahead for now

**Pull request merged**